src/button/button-group.scss

```diff
@@ -12,20 +12,18 @@
   > * {
     display: flex;
 
-    #{$childSelector} {
+    &:hover {
       position: relative;
+      z-index: 1;
+    }
 
-      &:hover {
-        z-index: 1;
-      }
-
-      &:focus {
-        z-index: 2;
+    &:focus-within {
+      position: relative;
```

Comment:
Why position is required here?

Reply:
Dont remember why it was needed but I've changed the approach now.